lib/debug/ProfilingPlugin.js

```diff
@@ -315,7 +315,7 @@ const interceptAllParserHooks = (moduleFactory, tracer) => {
 		moduleFactory.hooks.parser
 			.for(moduleType)
 			.tap("ProfilingPlugin", (parser, parserOpts) => {
-				interceptAllHooksFor(parser, tracer, "Parser");
+				interceptAllHooksFor(parser, tracer, "JavascriptParser");
```

Comment:
This should not be renamed as it points to all kind of parsers

Reply:
reverted